resources/views/layouts/components/header.blade.php

```diff
@@ -0,0 +1,43 @@
+<!-- preloader section start -->
+<div id="preloader">
+    <span class="loader"></span>
+</div>
+<!-- preloader section end -->
```

Comment:
Move this to **resources/views/layouts/app.blade.php**

Reply:
i move this preloader into app.blade.php